apps/keyboard/keyboard.properties.po

```diff
@@ -161,29 +161,29 @@ msgstr ""
 
 #: userDictionaryAddNow
 msgid "Add a word now."
-msgstr ""
+msgstr "Faka ibala khathesi"
 
 #: userDictionaryAddButton.ariaLabel
 msgid "add word"
-msgstr ""
+msgstr "faka ibala"
 
 #: userDictionaryAdd
 msgid "Add Word"
-msgstr ""
+msgstr "Faka ibala"
 
 #: userDictionaryEdit
 msgid "Edit Word"
-msgstr ""
+msgstr "Tshintsha Ibala"
 
 #: userDictionarySaveButton
 msgid "Save"
-msgstr ""
+msgstr "Gcina"
 
 #: userDictionaryDeleteButton
 msgctxt "userDictionaryDeleteButton"
 msgid "Delete"
-msgstr ""
+msgstr "Citsha"
 
 #: userDictionaryDeletePrompt
 msgid "Delete word “{{word}}”?"
-msgstr ""
+msgstr "Citsha ibala \"{{ibala}}\"?"
```

Comment:
the "\" should be removed from here since it doesn't appear in the original string